doozer/doozerlib/lock.py

```diff
@@ -0,0 +1,30 @@
+from multiprocessing import Semaphore
```

Comment:
This could also be part of `artcommonlib`

Reply:
makes sense. I will move it to artcommonlib.